examples/scatter-matrix-app.py

```diff
@@ -0,0 +1,42 @@
+from dash import Dash, dcc, html, Input, Output
+import pandas as pd
+import plotly.express as px
+
+filepath = "https://raw.githubusercontent.com/plotly/datasets/master/diabetes.csv"
+df = pd.read_csv(filepath)
+# fig = px.scatter_matrix(df)
```

Comment:
It's better to remove commented out code

Reply:
done!